src/unitxt/operators.py

```diff
@@ -977,7 +977,7 @@ def process(
             if self.process_every_value:
                 assert isinstance(
                     value, list
-                ), f"'process_every_value' can be set to True only for fields that contain lists, whereas in instance {instance}, the contents of field '{field_name}' is of type '{type(value)}'"
+                ), f"'process_every_value' can be set to True only for fields that contain lists, the contents of field '{field_name}' is of type '{type(value)}'"
```

Comment:
suggesting to use same phrasing as above in this case of `process_every_value`:
f"'process_every_field' == True is allowed only for fields whose values are lists, but value of field '{field_name}' is '{value}'"